posts/2022-10-27-22.0.0.12-beta.adoc

```diff
@@ -0,0 +1,351 @@
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+open-graph-image-alt: Open Liberty Logo
+---
+= More Jakarta EE 10 and MicroProfile 6 features make their debut in Open Liberty 22.0.0.12-beta!
+Michal Broz <https://github.com/mbroz2>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+
+As we continue to make progress towards full Jakarta EE 10 compliance, the 22.0.0.12-beta release brings with it six new Jakarta EE 10 specifications, alongside the ones which have been part of previous beta releases.  Jakarta Batch 2.1 provides improved CDI integration while Jakarta XML Web Services 4.0 now incorporates Jakarta Web Services Metadata.  Jakarta Pages 3.1 allows developers to identify undefined Expression Language variables and raise an Exception, while Jakarta Tags 3.0 includes Tag URI renaming when referencing tag libraries. Jakarta Messaging 3.1 makes connection factory and destination definition annotations repeatable, while Jakarta WebSocket 2.1 allows for the registration of endpoints outside of the deployment phase.
+
+But it's not only Jakarta EE 10 features that have made this beta, as two new MicroProfile 6 features are also included! MicroProfile JSON Web Token 2.1 brings with it simplified JWT validation and MicroProfile Metrics 5.0 provides user defined metric registry scopes.
+
+
+The link:{url-about}[Open Liberty] 22.0.0.12-beta includes the following beta features (along with link:{url-prefix}/docs/latest/reference/feature/feature-overview.html[all GA features]):
+
+* Jakarta EE 10:
+** <<pages, Identify undefined Expression Language variables and raise Exception (Jakarta Pages 3.1 and Jakarta Tags 3.0)>>
+** <<websocket, Register endpoints outside of the deployment phase (Jakarta WebSocket 2.1)>>
+** <<batch, Improved CDI integration (Jakarta Batch 2.1)>>
+** <<messaging, Connection factory and destination definition annotations are now repeatable (Jakarta Messaging 3.1)>>
+** <<xmlws, Jakarta Web Services Metadata (Jakarta XML Web Services 4.0)>>
+* MicroProfile 6:
+** <<jwt, Simplified JWT Validation (MicroProfile JSON Web Token 2.1)>>
+** <<metrics, User defined metric registry scopes (MicroProfile Metrics 5.0)>>
+
+
+See also link:{url-prefix}/blog/?search=beta&key=tag[previous Open Liberty beta blog posts].
+
+
+
+// // // // DO NOT MODIFY THIS COMMENT BLOCK <GHA-BLOG-TOPIC> // // // // 
+// Blog issue: https://github.com/OpenLiberty/open-liberty/issues/22563
+// Contact/Reviewer: volosied,pnicolucci,ReeceNana
+// // // // // // // // 
+[#pages]
+== Identify undefined Expression Language variables and raise Exception (Jakarta Pages 3.1 and Jakarta Tags 3.0)

```

Comment:
Is this it title document of just a highlight of new feature? 

Looking over it again, I would like to reword it.  Something like: `Option to Raise Exceptions for undefined Expression Language Identifiers"